app/models/bishop.rb

```diff
@@ -1,2 +1,10 @@
 class Bishop < ChessPiece
+  def display
+    if color == "black"
+    return "&#9821;"
+    else 
+      return "&#9815;"
+      
```

Comment:
We should clean up the extra white space and make sure the indentations are formatted correctly